src/components/user/ha-user-picker.ts

```diff
@@ -29,12 +29,12 @@ class HaUserPicker extends LitElement {
       return [];
     }
 
-    const filteredUsers = users!.filter((user) => !user.system_generated);
+    let sorted = users!.filter((user) => !user.system_generated);
 
-    if (filteredUsers.length === 1) {
-      return filteredUsers;
+    if (sorted.length === 1) {
+      return sorted;
     }
-    const sorted = [...filteredUsers];
+    sorted = [...sorted];
```

Comment:
This line can be removed

Reply:
We created a copy because we didn't want to sort the array that wasn't ours, but only for the rendering. Now that we have a `filter` that makes a copy for us, it is no longer needed.